pages/HygieneKitsCard.js

```diff
@@ -13,7 +13,7 @@ function HygieneKitsCard() {
           </p>
           <p>
             <a
-              class="btn btn-primary"
+              className="btn btn-primary"
```

Comment:
Good catch, thanks for noticing and fixing this! I really appreciate your attention to detail and code quality.

Since this project is still small, I'm good with accepting this even if outside the scope of the HotMealDayCard. When this project grows larger with more contributors, we may ask for improvements like this to be in a separate PR.